README.md

```diff
@@ -39,6 +39,3 @@ Front do projeto de Reconhecimento de produtos via YoLo
   ]
 }
 ```
-
-- Melhorias propostas
```

Comment:
@NicolasCendron removed as I was able to run this through localhost, without communicating through network at all.
Or did you meant something else by this?

Reply:
I mean it downloads jquery and bootstrap from CDNs on runtime.
If you don't have internet connection it will break